src/kakarot/kakarot.cairo

```diff
@@ -167,17 +167,18 @@ func set_native_token{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_chec
 }
 
 // @notice Deploy a new contract account and execute constructor
+// @param gas_limit: maximum amount of gas the constructor can use
 // @param bytes_len: the constructor + contract bytecode lenght
 // @param bytes: the constructor + contract bytecode
 // @return evm_contract_address The evm address that is mapped to the newly deployed starknet contract address
 // @return starknet_contract_address The newly deployed starknet contract address
 @external
 func deploy_contract_account{
     syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr, bitwise_ptr: BitwiseBuiltin*
-}(bytecode_len: felt, bytecode: felt*) -> (
+}(gas_limit: felt, bytecode_len: felt, bytecode: felt*) -> (
```

Comment:
I don't think we need gas_limit as a param, we decided in the community call to deprecate the use of gas_limit in our logic.
I think a pragmatic approach would be to remove this most recent addition of gas_limit inside the function

Reply:
Okidoki. Removing it.